follow/models.py

```diff
@@ -168,3 +168,13 @@ def cancel(self):
         request_cancel.send(sender=self)
         self.delete()
         return True
+
```

Comment:
Can you also scaffold `RemoteFollow` where we keep track of who our authors follow?

Reply:
Where did `RemoteFollower` go? (i.e. remote user following our author)

Reply:
This is the remoteFollower, sorry for confusing naming, I am currently working on remotefollowing